tests/order.py

```diff
@@ -1,8 +1,9 @@
 import json
-from datetime import date, datetime
+from datetime import date 
 from rest_framework import status
 from rest_framework.test import APITestCase
-
+from django.db import models
```

Comment:
I think these two imports were added when you were playing around with parsing the url string. If we don't need them, I say let's remove them!

Reply:
great point took those out